update: consolidate content in Monitoring (#131)
aiven/aiven-docs: docs/integrations/send-logs-to-elasticsearch.md

Start by setting up an external service integration for Elasticsearch.

1.  Log in to the [Aiven Console](https://console.aiven.io/).
1.  Go to **Service Integration** from the menu on the left.
1.  Select **External Elasticsearch** from the list.
1.  Select **Add new endpoint**.
1.  Set a preferred *endpoint name*, we'll call it `CONNECTION_NAME`
    later.
1.  In the connection URL field set the connection string in a format
    `https://ELASTICSEARCH_USER:ELASTICSEARCH_PASSWORD@ELASTICSEARCH_HOST:ELASTICSEARCH_PORT`,
    using your own values for those parameters.
1.  Set desired index prefix, that doesn't overlap with any of already
    existing indexes in your Elasticsearch service.

> 1.  If you need a certificate to access the endpoint, add the body of

Reviewer: ```suggestion
1.  Optional: Add the body of your CA certificate in PEM format. 
```

>     your CA certificate in PEM format. This field is optional.

Reviewer: This and the change before it are suggested to reduce word count and use Google's guide for optional steps.

```suggestion
```

> 1.  Set other fields based on your requirements, or leave the default

Reviewer: There are two optional fields pre-filled with default values. I'd personally probably just delete the step. 🤷

    values there.

> 1.  Select **Create**.

Reviewer: ```suggestion
1.  Click **Create**.
```

> A new service integration will be added. You can now reference your

Reviewer: Are either of these sentences needed? 🪓

service by the `CONNECTION_NAME` you chose.

> ## Send logs to an external service

Reviewer: Not sure, but perhaps "Send service logs to Elasticsearch" would be better when someone's searching the docs? The reason being that there are different types of logs in Aiven, and there's no longer an "internal" Elasticsearch service in Aiven.

> 1.  Go to **Services** from the menu on the left.

Reviewer: ```suggestion
1.  Go to **Services**.
```

> 1.  Select the service which logs to send to the external

Reviewer: Maybe just: `Select a service.`?

    Elasticsearch service.

> 1.  On the **Overview** page of your service, go to the **Service

Reviewer: Very minor, but I'd suggest getting rid of this and the next step in lieu of simply: `1. Click **Integrations**.`

It removes the need for the user to scroll and find the service integrations section so it's a more user-friendly way of getting to the same thing, but it's also less likely to change than the overview page itself (I _think_ 😁 ).

    integrations** section.
1.  Select **Manage integrations**.
1.  Select **Elasticsearch Logs** from the list.

> 1.  In the newly appeared modal window, select the endpoint with name

Reviewer: ```suggestion
1.  Select the **Endpoint name**
```

>     `CONNECTION_NAME` from the list and select **ENABLE**. Close the

Reviewer: ```suggestion
    and click **Enable**.
```

    modal window.

> 1.  Observe the status change for newly added integration in the

Reviewer: Not sure this is the best way to word this, but again there's no need to go to the Overview page (probably had to in the old UI). I usually put these types of things at the end instead of as a separate step, but no strong opinions about that. 🤔 

```suggestion
You can view the status of the integration at the top of the page.
```

    **Service integrations** section on the **Overview** page of your
    service.

> 1.  Verify that the logs are flowing into your Elasticsearch.

Reviewer: IMHO, this step doesn't add any value.
